inc/html.class.php

```diff
@@ -138,7 +138,7 @@ static function entities_deep($value) {
    **/
    static function convDate($time, $format = null) {
 
-      if (is_null($time) || $time == 'NULL' || trim($time) == '') {
+      if (is_null($time) || $time == 'NULL' || trim($time) == '' || $time == '0000-00-00 00:00:00') {
```

Comment:
```suggestion
      if (is_null($time) || trim($time) == '' || in_array($time, ['NULL', '0000-00-00', '0000-00-00 00:00:00'])) {
```

Reply:
If we filter on `'0000-00-00 00:00:00'`, we should also filter on `'0000-00-00'`, as this method can also be used to format `date` values (i.e. https://github.com/glpi-project/glpi/blob/9.4/bugfixes/inc/infocom.class.php#L421).